controller/registry/private.py

```diff
@@ -87,8 +87,7 @@ def _put_first_image(repository_path):
 
 
 def _api_call(endpoint, data=None, headers={}, cookies=None, request_type='GET'):
-    # FIXME: update API calls for docker 0.10.0+
-    base_headers = {'user-agent': 'docker/0.9.0'}
+    base_headers = {'user-agent': 'docker/0.10.1'}
```

Comment:
I guess docker/1.0.0 might be a bit more future-proof. I'll give it a try and update this PR if that User-Agent string works.